readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
We also need the ability for a JS function to return a subarray of ISTF tokens, not just a single value. You might have mixins for things like media queries that return a whole block of CSS.